test/adapter/json_api/parse_test.rb

```diff
@@ -125,7 +125,7 @@ def test_polymorphic
               src: 'http://example.com/images/productivity.png',
               author_id: nil,
               photographer_id: '9',
-              photographer_type: 'people',
```

Comment:
n/m I take that back.  You're right. I misread that this was the input being changed, not the output.  Rails needs this format as the deserialized attributes.